Azure/azure-cli
# [Privatedns] Add feature in private DNS zone to import export zone file

**Description**  

This PR contains the changes for importing and exporting zone files to private dns zone. This is added as new feature in private dns zone. This feature was already present in public Azure DNS, incorporating similar changes for Private DNS zone.

**Testing Guide**  

az network private-dns zone import -g MyResourceGroup -n MyZone -f /path/to/zone/file

 az network private-dns zone export -g MyResourceGroup -n www.mysite.com -f mysite_com_zone.txt

---

This checklist is used to make sure that common guidelines for a pull request are followed.
